tests/integration/models/release.spec.js

```diff
@@ -276,6 +276,44 @@ describe('Release Model', function () {
 					});
 				});
 			});
+
+			describe('balena.model.release.setIsInvalidated()', function () {
+				releaseRetrievalFields.forEach((field) => {
+					parallel('', function () {
+						it(`should validate a release by ${field}`, async function () {
```

Comment:
This release was already valid even before the test :)
We might better move it below the other.